crates/nvngx-sys/Cargo.toml

```diff
@@ -24,6 +24,7 @@ cc = "1"
 [dependencies]
 widestring = "1"
 libc = "0.2"
+ash = { version = "0.38", default-features = false }
```

Comment:
In the future, I was thinking of letting the users decide which vulkan crate to have a dependency on in their dependency tree. So that I would probably have tried to move the ash implementation into a separate module, and provide implementations for the other vulkan crates as well. Of course, with a proper abstraction layer based on traits, or something like that. Do you think it makes sense? Will you concentrate on ash, by the way?

Reply:
At least this PR already makes it easier to replace that by simply `cfg`'ing or otherwise adjusting the `use`s in `lib.rs` that import the `Vk` types into place.

However I don't think there are currently that many alternatives available if at all, nor think there should be at least for `-sys` bindings which will only make things **less compatible**. `ash` is "unfortunately" in a bit of a weird place where it provides `-sys`-like bindings with a few higher-level abstractions on top, created before `-sys` crates were really a thing (and well before I learned Rust).

> Will you concentrate on ash, by the way?

What do you mean here?